API.md

```diff
@@ -1,64 +1,272 @@
+    - [expectContainsJSON([path], json)](#expectcontainsjsonpath-json)
+    - [expectJSONTypes([path], schema)](#expectjsontypespath-schema)
+    - [expectBodyContains(content)](#expectbodycontainscontent)
+    - [expectJSONLength([path], length)](#expectjsonlengthpath-length)
+    - [expectMaxResponseTime(ms)](#expectmaxresponsetimems)
+    - [not()](#not)
+    - [Using Paths](#using-paths)
+      - [Testing Nested Objects](#testing-nested-objects)
+      - [Testing All Objects in an Array](#testing-all-objects-in-an-array)
+      - [Testing One Object in an Array](#testing-one-object-in-an-array)
+  - [useApp()](#useapp)
+    - [Example Use:](#example-use)
+      - [Express Application:](#express-application)
+      - [IcedFrisby Test:](#icedfrisby-test)
+  - [Global Setup](#global-setup)
+    - [request.baseUri](#requestbaseuri)
+    - [request.headers](#requestheaders)
+    - [request.json](#requestjson)
+    - [request.inspectOnFailure](#requestinspectonfailure)
+    - [failOnMultiSetup](#failonmultisetup)
+    - [useApp](#useapp-in-globalsetup)
+    - [Resetting `globalSetup`](#resetting-globalsetup)
+  - [Helpers](#helpers)
+    - [before()](#before)
+    - [after()](#after)
+    - [finally()](#finally)
+    - [afterJSON()](#afterjson)
+    - [only()](#only)
+    - [timeout(ms)](#timeoutms)
+    - [retry(count, backoff)](#retrycount-backoff)
+    - [baseUri(uri)](#baseuriuri)
+    - [waits(ms)](#waitsms)
+    - [exceptionHandler(function)](#exceptionhandlerfunction)
+  - [Inspectors](#inspectors)
+    - [inspect(cb)](#inspectcb)
+    - [inspectRequest(message)](#inspectrequestmessage)
+    - [inspectResponse(message)](#inspectresponsemessage)
+    - [inspectHeaders(message)](#inspectheadersmessage)
+    - [inspectJSON(message)](#inspectjsonmessage)
+    - [inspectBody(message)](#inspectbodymessage)
+    - [inspectStatus(message)](#inspectstatusmessage)
+    - [Send Raw JSON or POST Body](#send-raw-json-or-post-body)
+
+## The Basics
+
+Every frisby request begins with `create(..)` and ends with a `toss()`.
+
+### create(msg)
+
+- Types: `msg`: `string`
+- Default: `none`
+
+Used to create an instance of IcedFrisby that is used to send 1 request and receive the response.
+The `msg` is used to name the test when it's wrapped in mocha for execution at runtime.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+### toss()
+
+Used to complete the list of commands and issue the request.
+
+```javascript
+frisby.create('a test')
+    .get('http://example.com')
+    .expectStatus(200)
+    // any number of additional expect statements here
+    .toss();
+```
+
+## Commands
+
+These are the commands you'll need to get IcedFrisby talking HTTP.
+
+An optional parameters object is accepted in all of these methods, but aren't _useful_ for every verb. All object parameters are optional.
```

Comment:
"An optional `params` object is accepted by each of these methods" would be a little bit clearer. The params are useful for every verb, so I'd delete the second half of the first sentence.

Reply:
With the exception of the mock, they're only useful for POST/PUT/PATCH, but you're right - it's disingenuous. I'll fix it.